Sources/SwifterSwift/UIKit/UITextFieldExtensions.swift

```diff
@@ -172,6 +172,18 @@ public extension UITextField {
         rightView = iconView
         rightViewMode = .always
     }
+    
+    /// Add tool bars to the textfield input accessory view.
+    /// - Parameters:
+    ///   - items: Pass UIBarButtonItems.
```

Comment:
Remove the word "Pass" for both of these params and give a more detailed description. E.g.
```
- items: The items to present in the toolbar.
- height: The height of the toolbar.
```

Reply:
Done